.travis.yml

```diff
@@ -1,13 +1,18 @@
 language: go
-script: "./test.sh"
 go:
-    - 1.5
-    - 1.6
-    - 1.7
-    - tip
+  - 1.5.4
+  - 1.6.3
```

Comment:
Any reason why you would like to specify the go patch level? As far as I understand [gimme](https://github.com/travis-ci/gimme) will provide the latest patch level if it's not provided.